stories/Atom/BaseTypography/Abbr/Abbr.jsx

```diff
@@ -1,9 +1,10 @@
 import React from 'react';
 import './abbr.scss';
 
-export const Abbreviation = ({ label }) => (
-  <abbr>
-    { label }
-  </abbr>
-);
-
+export function Abbreviation({ label }) {
+  return (
```

Comment:
What's the reason for this code change? Linting?

Reply:
I haven't worked on a code change here:

Screenshot:

<img width="1200" alt="Screenshot 2023-01-06 at 10 59 19 AM" src="https://user-images.githubusercontent.com/12831545/210999369-da031f29-0bb9-43bb-8b9b-ca168d05e1ad.png">